modules/ROOT/partials/configuration/valid_elements.adoc

```diff
@@ -3,17 +3,18 @@
 
 The valid_elements option defines which elements will remain in the edited text when the editor saves. You can use this to limit the returned HTML to a subset.
 
-This option contains a comma separated list of element conversion chunks. Each chunk contains information about how one element and its attributes should be treated. The default rule set for this option is a mixture of the full https://html.spec.whatwg.org/[HTML5] and http://www.w3.org/TR/REC-html40/[HTML4] specification or the HTML5 or HTML4 specification depending on the configured link:{baseurl}/configure/content-filtering/#schema[schema].
+This option contains a comma separated list of element conversion chunks. Each chunk contains information about how one element and its attributes should be treated. The default rule set for this option is a mixture of the full https://html.spec.whatwg.org/[HTML5] and http://www.w3.org/TR/REC-html40/[HTML4] specification or the HTML5 or HTML4 specification depending on the configured xref:schema[`schema`].
 
-If you just want to add or change some behavior for a few items, use the link:{baseurl}/configure/content-filtering/#extended_valid_elements[extended_valid_elements] option
+If you just want to add or change some behavior for a few items, use the xref:extended_valid_elements[`extended_valid_elements`] option
 
+[discrete]
 === Control characters
 
 |===
 | Name | Summary
 
 | @
-| Rules with this name will be applied to all elements defined after this rule. So `+@[attr1&#124;attr2]+` will enable `attr1` and `attr2` for all elements, but `+element1,@[attr1&#124;attr2],element2,element3+` will enable `attr1` and `attr2` only for `element2` and `element3`. If applied in link:{baseurl}/configure/content-filtering/#extended_valid_elements[extended_valid_elements], it is only effective for the elements in the extended_valid_elements list.
+| Rules with this name will be applied to all elements defined after this rule. So `+@[attr1&#124;attr2]+` will enable `attr1` and `attr2` for all elements, but `+element1,@[attr1&#124;attr2],element2,element3+` will enable `attr1` and `attr2` only for `element2` and `element3`. If applied in xref:extended_valid_elements[`extended_valid_elements`], it is only effective for the elements in the extended_valid_elements list.
```

Comment:
```suggestion
| Rules with this name will be applied to all elements defined after this rule. So `+@[attr1\|attr2]+` will enable `attr1` and `attr2` for all elements, but `+element1,@[attr1\|attr2],element2,element3+` will enable `attr1` and `attr2` only for `element2` and `element3`. If applied in xref:extended_valid_elements[`extended_valid_elements`], it is only effective for the elements in the extended_valid_elements list.
```
This isn't rendering correctly (though it also appears to be an issue in production today as well).